### PR TITLE
Feature/camera settings

### PR DESCRIPTION
**Pull request by stfuchs**
### make the color camera settings available in libfreenect2::Frame.
see #156

### CODE REVIEW

File: include/libfreenect2/frame_listener.hpp [hunk 1]
```diff
@@ -54,11 +54,17 @@ class LIBFREENECT2_API Frame
...
+  float exposure;         ///< Get the exposure time set by the color camera.
+  float gain;             ///< Get the gain set by the color camera.
+  float gamma;            ///< Get the gamma level set by the color camera.
 
   Frame(size_t width, size_t height, size_t bytes_per_pixel) :
     width(width),
     height(height),
-    bytes_per_pixel(bytes_per_pixel)
+    bytes_per_pixel(bytes_per_pixel),
+    exposure(30.f),
```
Comment (floe) [inline include/libfreenect2/frame_listener.hpp]: If this is in milliseconds, should the default value then be 33.3f?

Comment (stfuchs) [inline include/libfreenect2/frame_listener.hpp]: To be honest, I have no idea what might be a good default value. It ranges from < 0.1 up to 66.4719085693 when it's fully covered. 

So you mean `1.f/30.f * 1000.f`?

Comment (floe) [inline include/libfreenect2/frame_listener.hpp]: Yes, something along those lines. 66.47 in milliseconds would be 15 Hz, and I remember something that the color camera does indeed drop the framerate to 15 Hz in low-light situations. So default at 30 Hz would then be ~ 33.3.

Comment (xlz) [inline include/libfreenect2/frame_listener.hpp]: It should be 0.0f or NaN? There is no gamma for IR or depth images.

Comment (stfuchs) [inline include/libfreenect2/frame_listener.hpp]: Agree. But 1. would be the identity transfer function for gamma and should also work fine as default right?

Comment (xlz) [inline include/libfreenect2/frame_listener.hpp]: Basically give it in invalid value because it will be filled in if working correctly.

Comment (stfuchs) [inline include/libfreenect2/frame_listener.hpp]: Ok. NaN would require `#include <limits>`, is that fine with you?

Comment (xlz) [inline include/libfreenect2/frame_listener.hpp]: If you ask I'd go with 0.0f.

**Comment by floe** [comment]
I'll merge this for 0.1, as the API remains unchanged (even though it will break the ABI AFAICT).

**Comment by xlz** [comment]
Can this be merged? The documentation PR is after this.

**Comment by floe** [comment]
Ah, sure, had sort of forgotten about that one, sorry.

**Pull request merged**